javascript/packages/orchestrator/src/paras-decorators/moonbeam.ts

```diff
@@ -52,13 +52,20 @@ async function addAuthority(specPath: string, node: Node, key: GenesisNodeKey) {
 
   keys.push(key);
 
-  console.log(
-    `\t👤 Added Genesis Authority ${decorators.green(
-      node.name,
-    )} - ${decorators.magenta(sr_account.address)}`,
-  );
+  new CreateLogTable({
+    colWidths: [30, 20, 70],
+  }).pushToPrint([
+    [
+      decorators.cyan("👤 Added Genesis Authority"),
+      decorators.green(node.name),
+      decorators.magenta(sr_account.address),
+    ],
+  ]);
+
+  new CreateLogTable({
+    colWidths: [120],
+  }).pushToPrint(chainSpec.genesis.runtime.authorMapping);
```

Comment:
```suggestion
  }).pushToPrint([chainSpec.genesis.runtime.authorMapping]);
```
The ci is failing, I think this is the cause.

Thanks!